apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/stu3/providers/TransformerUtils.java

```diff
@@ -3364,30 +3363,21 @@ public static Bundle addResourcesToBundle(Bundle bundle, List<IBaseResource> res
       }
     }
 
-    logBeneIdToMdc(beneIds);
+    logBeneIdToMdc(beneIds.stream().map(Long::valueOf).toArray(Long[]::new));
 
     return bundle;
   }
 
   /**
    * Output list of benefificiary IDs to MDC logging
    *
-   * @param beneIds the {@link Collection} of beneficiary IDs top log
+   * @param beneIds the {@link Long} of beneficiary IDs top log
    */
-  public static void logBeneIdToMdc(Collection<String> beneIds) {
-    if (!beneIds.isEmpty()) {
-      MDC.put("bene_id", String.join(", ", beneIds));
-    }
-  }
-
-  /**
-   * Output beneficiary ID to the MDC logging
-   *
-   * @param beneId the {@link Long} beneficiary ID to log
-   */
-  public static void logBeneIdToMdc(Long beneId) {
-    if (beneId != null) {
-      MDC.put("bene_id", String.valueOf(beneId));
+  public static void logBeneIdToMdc(Long... beneIds) {
+    if (beneIds.length > 0) {
+      String beneIdEntry =
+          Arrays.stream(beneIds).map(String::valueOf).collect(Collectors.joining(", "));
+      MDC.put("bene_id", beneIdEntry);
```

Comment:
I think we should take this opportunity to move this helper methods out of V1/V2 specific classes and reduce the duplication. We only need one copy of this method.

Reply:
Put method in Commons